src/Orc.Controls/Orc.Controls.Shared/Controls/DateRangePicker/Views/DateRangePickerControl.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<catel:UserControl x:Class="Orc.Controls.DateRangePickerControl"
+                   xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                   xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+                   xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                   xmlns:controls="clr-namespace:Orc.Controls"
+                   xmlns:converters="clr-namespace:Orc.Controls.Converters"
+                   xmlns:catel="http://catel.codeplex.com"
+                   xmlns:sys="clr-namespace:System;assembly=mscorlib"
+                   mc:Ignorable="d" x:Name="CurrentControl" d:DesignHeight="22" d:DesignWidth="190">
+
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <Label Grid.Row="0" Grid.Column="0" Content="{catel:LanguageBinding ResourceName=Controls_DateRangePicker_Start_Lbl}"/>
+        <controls:DateTimePickerControl Grid.Row="0" Grid.Column="1" 
+            Value="{Binding DateStart}"
+            AllowNull="False"
+            AllowCopyPaste="{Binding DateTimePickerAllowCopyPaste, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type controls:DateRangePickerControl}}}"
```

Comment:
Why not use AllowCopyPaste, etc? I don't see the added value of the DateTimePicker prefix.